index.html

```diff
@@ -4939,13 +4939,16 @@ <h3>Base Context</h3>
           <p>
 Implementations MUST ensure that the base context value, located at
 <code>https://www.w3.org/ns/credentials/v2</code>, matches the following
-SHA-256 digest of the value:
-<strong><code>dc28cf86d8a3199beb9d14e29d1da662346b8660abf2edca873098c88ca2a457</code></strong>.
+SHA-384 digest of the value computed and encoded according to the [[SRI]] definition of `digest`:
+<strong><code>lHKDHh0msc6pRx8PhDOMkNtSI8bOfsp4giNbUrw71nXXLf13nTqNJoRp3Nx+ArVK</code></strong>.
 The base context value matching the digest previously stated can be used to
 implement a local cached copy. It is possible to confirm the
-SHA-256 digest by running the following command from a modern Unix command
+SHA-384 digest by running the following command from a modern Unix command
 interface line:
-`curl -s https://www.w3.org/ns/credentials/v2 | openssl dgst -sha256 -hex -r`.
+`curl -s https://www.w3.org/ns/credentials/v2 | openssl dgst -sha384 -binary | openssl base64 -A`.
+
+More details regarding this hash encoding and methods may be found in the <a href="https://www.w3.org/TR/SRI/#integrity-metadata">integrity metadata</a> 
```

Comment:
This sentence puzzles me. `this hash encoding and methods` might should be `these hash encoding and methods` or `this hash encoding method` or something else. I'm guessing the latter, but could be convinced otherwise.
```suggestion
More details regarding this hash encoding method may be found in the <a href="https://www.w3.org/TR/SRI/#integrity-metadata">integrity metadata</a> 
```

Reply:
totally fine with your suggestion here

Reply:
This needs to be committed before the PR is merged...